databases/week3/make_database.sql

```diff
@@ -0,0 +1,38 @@
+CREATE DATABASE hyf_meal;
+USE hyf_meal;
+CREATE TABLE meal
+(
+	id INT UNSIGNED AUTO_INCREMENT NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    description TEXT,
+    location VARCHAR(255) NOT NULL,
+    event_time DATETIME NOT NULL, -- CHECK( event_time > CURRENT_TIMESTAMP() ) DEFAULT(getdate()),
+    max_reservation INT UNSIGNED NOT NULL DEFAULT('1'),
+    price DECIMAL NOT NULL,
```

Comment:
Decimal without any arguments will actually default to `DECIMAL(10, 0)`, which will not allow you to store any digits after the point, fx `99.95`
For prices and other monetary values i would recommend `DECIMAL(13, 4)` instead. See: https://rietta.com/blog/best-data-types-for-currencymoney-in/